.pylintrc-mandatory.jinja

```diff
@@ -3,19 +3,41 @@ load-plugins=pylint_odoo
 score=n
 
 [ODOOLINT]
+{%- if odoo_version < 16 %}
 readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
 manifest_required_keys=license
 manifest_deprecated_keys=active
 valid_odoo_versions={{ odoo_version }}
+{%- else %}
+readme-template-url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest-required-keys=license
+manifest-deprecated-keys=active
+valid-odoo-versions={{ odoo_version }}
+{%- endif %}
```

Comment:
I don't think this depends on the odoo_version. This really depends on the pylint_odoo version, since at some point they changed the keys from having underscores (old version) to having dashes (new version) to better follow conventions

Reply:
The problem is that the pylint-odoo version is not known at this point. But you are right, it would be clearer if it was compared with the pylint-odoo version.
In the end, the pylint-odoo version used depends on the Odoo version. So, it can be used if you know which version you are using.